src/EditorFeatures/Core/Shared/Tagging/Utilities/TagSpanIntervalTree.TagNode.cs

```diff
@@ -1,37 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Microsoft.VisualStudio.Text;
-using Microsoft.VisualStudio.Text.Tagging;
-
-namespace Microsoft.CodeAnalysis.Editor.Shared.Tagging;
-
-internal partial class TagSpanIntervalTree<TTag>
-{
-    private readonly struct TagNode(ITagSpan<TTag> tagSpan, SpanTrackingMode trackingMode)
```

Comment:
no need to store the trackingMode in the node itself.  all nodes within the TagSpanIntervalTree have hte same tracking mode.  So this was redundant.  at that point, this entire type was redundant as it was just wrapping the ITagSpan<TTag>